frontend/src/components/data-table/url-detector.tsx

```diff
@@ -1,12 +1,14 @@
 /* Copyright 2024 Marimo. All rights reserved. */
 
+import { marked } from "marked";
 import { useState } from "react";
 import {
   Popover,
   PopoverContent,
   PopoverTrigger,
 } from "@/components/ui/popover";
 import { Events } from "@/utils/events";
+import { MarkdownRenderer } from "../chat/markdown-renderer";
```

Comment:
we should move the markdown stuff to a common folder so data-table does not depend on the `chat/` folder